routes/user.js

```diff
@@ -38,8 +35,17 @@ router.get('/users/:id', async (req, res) => {
 router.delete('/users/:id', async (req, res) => {
   try {
     const _id = req.params.id;
-    const user = await UserModel.delete(_id);
-    res.status(200).send(user);
+    const user = await models['User'].findByPk(_id);
+    if (user) {
+      await models['User'].destroy({
+        where: {
+          id: _id,
+        },
+      });
+      res.status(200).send(user);
+    } else {
+      res.status(404).send(e);
```

Comment:
`e` is not defined I suppose

maybe we should use multiple `catch` blocks in this route (each for a situation that could throw)?